mods/ra/maps/fort-lonestar/fort-lonestar.lua

```diff
@@ -12,90 +6,62 @@ OilDerricks = { OilDerrick1, OilDerrick2, OilDerrick3, OilDerrick4 }
 SpawnPoints = { Spawn1, Spawn2, Spawn3, Spawn4 }
 Snipers = { Sniper1, Sniper2, Sniper3, Sniper4, Sniper5, Sniper6, Sniper7, Sniper8, Sniper9, Sniper10, Sniper11, Sniper12 }
 
+if Map.Difficulty == "1 Player" then
```

Comment:
Wouldn't it be better to count the actual number of players instead of forcing people to select the right option?